README.md

```diff
@@ -161,6 +161,12 @@ Flag to enable automation of `com.apple.security.application-groups` in entitlem
 Allowed values: `true`, `false`.
 Default to `true`.
 
+`requirements` - *String*
+
+Specify the criteria that you recommend to be used to evaluate the code signature
+.
```

Comment:
Hi @savanain, sorry for my late reply. Would you mind moving this period to the line above? 🎨